src/dist/triple.rs

```diff
@@ -1,51 +1,9 @@
-use once_cell::sync::Lazy;
-use regex::Regex;
+mod known;
 
-// These lists contain the targets known to rustup, and used to build
-// the PartialTargetTriple.
+pub use self::known::{LIST_ARCHS, LIST_ENVS, LIST_OSES};
```

Comment:
Would be nice to split this commit in two parts: one that moves the constants to a separate file and another that starts using them in `toolchain::names`.

We don't need to redirect via `self::` here, we can import from `known::` directly.